application/src/main/java/org/togetherjava/tjbot/commands/tag/TagManageCommand.java

```diff
@@ -40,21 +33,21 @@ public TagManageCommand(TagSystem tagSystem) {
     public @NotNull CommandData addOptions(@NotNull CommandData commandData) {
         return commandData.addSubcommands(
                 new SubcommandData("rawtag", "View a tag in its raw form")
-                    .addOption(OptionType.STRING, "id", TAG_ID),
+                    .addOption(OptionType.STRING, "id", "tag id"),
                 new SubcommandData("edittag", "Edit a tag")
-                    .addOption(OptionType.STRING, "id", TAG_ID, true)
-                    .addOption(OptionType.STRING, CONTENT, "Tag content", true),
+                    .addOption(OptionType.STRING, "id", "tag id", true)
+                    .addOption(OptionType.STRING, "content", "Tag content", true),
                 new SubcommandData("editidtag", "Edit a tag using a message id")
-                    .addOption(OptionType.STRING, TAG_MINUS_ID, TAG_ID, true)
-                    .addOption(OptionType.NUMBER, MESSAGE_MINUS_ID, "Message id", true),
+                    .addOption(OptionType.STRING, "tag-id", "tag id", true)
+                    .addOption(OptionType.NUMBER, "message-id", "Message id", true),
                 new SubcommandData("deletetag", "Delete a tag").addOption(OptionType.STRING, "id",
-                        TAG_ID, true),
+                        "tag id", true),
                 new SubcommandData("createtag", "Create a tag")
-                    .addOption(OptionType.STRING, "id", TAG_ID, true)
-                    .addOption(OptionType.STRING, CONTENT, "Tag content", true),
+                    .addOption(OptionType.STRING, "id", "tag id", true)
+                    .addOption(OptionType.STRING, "content", "Tag content", true),
                 new SubcommandData("createidtag", "Create a tag using a message id")
-                    .addOption(OptionType.STRING, TAG_MINUS_ID, TAG_ID, true)
-                    .addOption(OptionType.NUMBER, MESSAGE_MINUS_ID, "Message id", true));
+                    .addOption(OptionType.STRING, "tag-id", "tag id", true)
+                    .addOption(OptionType.NUMBER, "message-id", "Message id", true));
```

Comment:
I think using constants is better than having it with plain `String`s. That way we at least have compile-time safety.
Maybe you misunderstood my last comment, it was really only about the names of those constants. Please revert.

Reply:
done